### DISCUSSION TITLE
i download local wp and i think it change something on computer and my next js code font work with npm run dev

### DISCUSSION THREAD

**Discussion by edanTavory** [vercel/community#1200] [Help]
### Summary

to do npm run dev with error that i have

### Additional information

error - node_modules\next\dist\client\link.js (269:22) @ LinkComponent
error - Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor

**Comment by amyegan** [comment]
Sorry you're running into an error @edanTavory. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), or your repo if it is public, so I can get a better look at the source of the problem?
